Makefile

```diff
@@ -147,7 +147,7 @@ run: operator-sdk ## Run the compliance-operator locally
 	WATCH_NAMESPACE=$(NAMESPACE) \
 	KUBERNETES_CONFIG=$(KUBECONFIG) \
 	OPERATOR_NAME=compliance-operator \
-	$(GOPATH)/bin/operator-sdk up local --namespace $(NAMESPACE)
+	$(GOPATH)/bin/operator-sdk run local --namespace $(NAMESPACE)
```

Comment:
Looks like this change is not sufficient, you'll probably want to amend the line with this:
```
diff --git a/Makefile b/Makefile
index 0796b4d..58e4fbf 100644
--- a/Makefile
+++ b/Makefile
@@ -147,7 +147,7 @@ run: operator-sdk ## Run the compliance-operator locally
        WATCH_NAMESPACE=$(NAMESPACE) \
        KUBERNETES_CONFIG=$(KUBECONFIG) \
        OPERATOR_NAME=compliance-operator \
-       $(GOPATH)/bin/operator-sdk run local --namespace $(NAMESPACE)
+       $(GOPATH)/bin/operator-sdk run --local --namespace $(NAMESPACE)
 
 .PHONY: clean
 clean: clean-modcache clean-cache clean-output ## Clean the golang environment
```

Reply:
Fixed. Thanks!